gbfs.md

```diff
@@ -139,10 +139,9 @@ Publishers SHOULD implement auto-discovery of GBFS feeds by linking to the locat
 
 ### Localization
 
-* Each set of data files SHOULD be distributed in a single language as defined in `system_information.json`.
-* A system that wants to publish feeds in multiple languages SHOULD do so by publishing multiple distributions, such as:
-    * `https://www.example.com/data/en/system_information.json`
-    * `https://www.example.com/data/fr/system_information.json`
+* Each supported language MUST be listed in the `languages` field in `system_information.json`.
+* Translations SHOULD be provided for each supported language for all translateable fields.
```

Comment:
Because supported languages are a MUST but individual translations are a SHOULD, do you have any thoughts on preferred behavior when a string translation is missing for a particular language?  Similarly, should data be considered invalid if a translation is provided for an unsupported language?

Reply:
I agree. This should be a MUST as well.

Reply:
"Similarly, should data be considered invalid if a translation is provided for an unsupported language?" not sure if that is necessary - what purpose would it serve to specify this?

Reply:
The purpose would be to make it explicit that there should be 1:1 mapping between values in the `languages` field in `system_information.json` and individual localized strings in translatable fields.  I think changing this to MUST helps, but I'd go further (see comment below)